### DISCUSSION TITLE
unreal engine error

### DISCUSSION THREAD

**Discussion by SJ824** [microsoft/AirSim#4873] [General]
I am using ubuntu 20.04 and am currently trying to run airsim through unreal engine 4.270 version. I have copied the plugins file from airsim into a new unreal engine project file that I have created. I was hoping that this would automatically plug in airsim, but when I tried to open the .uproject file of my new project with unreal engine editor, I got the following message: 'engine modules are out of date and cannot be compiled while the engine is running. Please build through your IDE." I got the following error: "engine modules are out of date and cannot be compiled while the engine is running." How can I troubleshoot this?

I've done the initial setup and the unreal engine editor works fine, but I don't know how to get airsim to run initially.

**Comment by ghost** [comment]
You have to create a class and build the file

**Reply by SJ824** [reply]
How should I run it?

**Comment by ghost** [comment]
Check the documentation, it is given step by step in it
